plugins/package-managers/carthage/src/test/kotlin/CarthageTest.kt

```diff
@@ -128,4 +127,4 @@ class CarthageTest : WordSpec() {
 }
 
 private fun Carthage.resolveDependencies(definitionFile: File) =
-    resolveDependencies(USER_DIR, definitionFile, Excludes.EMPTY, AnalyzerConfiguration(), emptyMap())
+    resolveDependencies(definitionFile.parentFile, definitionFile, Excludes.EMPTY, AnalyzerConfiguration(), emptyMap())
```

Comment:
This functions seems a bit redundant with `resolveSingleProject()`. Would it make sense to drop it in favor of `resolveSingleProject()` maybe in a preceeding commit?